### DISCUSSION TITLE
SOLID for Raspberry Pi 4に含まれるOSSとは、何でしょうか？

### DISCUSSION THREAD

**Discussion by asano-tk** [KyotoMicrocomputer/solid-rapi4#2] [Q&A] [answered]
SOLID for Raspberry Pi 4の紹介ページに、「SOLID for Raspberry Pi 4に含まれるOSS(Open Source Software)のご使用にあたっては、ユーザーご自身の責任にて各OSSのルールに従ってください。」とありますが、具体的に何のOSSが含まれているでしょうか？

githubのドキュメントも読んだのですが、TOPPERS OS以外のものが分からなかったので、ご質問させていただきました。
明記されている場所などでも結構ですので、何かご回答いただけると助かります。

**Comment by kmc-kk** [comment, marked as answer]
### Raspberry Pi 4 上で動作するソフトウェア

Linuxシステムとして`Raspbery Pi OS`、リアルタイムOSとして`TOPPERS/FMP3`を使用しています。

- [Raspberry Pi OS](https://www.raspberrypi.com/software/)
- [TOPPERS/FMP3](https://www.toppers.jp/fmp3-kernel.html) 

また、[`Ghostd`](https://github.com/KyotoMicrocomputer/solid-rapi4/blob/main/doc/system-linux.md#ghostd), [`Ghost CLI`](https://github.com/KyotoMicrocomputer/solid-rapi4/blob/main/doc/system-linux.md#ghost-cli) で使用されているOSSは [Third Party Licenses](https://solid.kmckk.com/rpi4/doc/oss/copyright-3rdparty-crates.html) を参照してください。

### PC上で動作するソフトウェア

Windows PC上で動作するIDEやデバッガ、ツールチェーンなどで使用されているOSS情報については[ `Open Source Software used in SOLID`](https://solid.kmckk.com/SOLID/doc/latest/oss/index.html#open-source-software-used-in-solid) を参照してください。

**Reply by asano-tk** [reply]
ご回答ありがとうございます！
それぞれ、確認したいと思います。
